### ISSUE TITLE
[Bug] CKAN not updating repositories

### ISSUE THREAD

**Issue by theonegalen** [KSP-CKAN/CKAN#3743]
Background
----------
* Operating System: Windows 10
* CKAN Version: 1.31.2 "Juno"
* KSP Version: 1.12.4

Problem
-------
CKAN isn't updating the repositories on one of my game instances. It works on my JNSQ instance, but not my stock-size instance. Before it stopped updating on the stock-size instance, I was given the "this mod's metadata has been updated, would you like to reinstall it?" message and clicked yes. Since doing this, it will not update the repositories, either when I start CKAN completely after turning it off or when I click the manual refresh button. The client simply says "Repositories already up to date" in the status bar at the bottom.

EDIT: The timestamp on the registry.json file for the stock size instance was current, but only after copying the registry.json from the JNSQ instance to the stock size instance did it actually update the repositories. Of course that meant CKAN thought the wrong things were installed, so I had to revert and export my modlist.

**Comment by HebaruSan** [comment]
Duplicate of #3699

**Comment by HebaruSan** [comment]
Thanks for the report, that sounds pretty close to the sequence I guessed in <https://github.com/KSP-CKAN/CKAN/issues/3699#issuecomment-1345676281>. I would suggest trying the current dev build to get the fix for #3707, then try refreshing again after a day or two to ensure the server etag has changed:

- https://ksp-ckan.s3-us-west-2.amazonaws.com/ckan.exe
